ods_ci/tests/Tests/400__ods_dashboard/430__data_science_pipelines/434__data-science-pipelines-ui.robot

```diff
@@ -22,7 +22,17 @@ Verify User Can Create, Run and Delete A DS Pipeline From DS Project Details Pag
     [Documentation]    Verifies user are able to create and execute a DS Pipeline leveraging on
     ...                DS Project UI
     [Tags]    Sanity    Tier1
-    ...       ODS-2206    ODS-2207
+    ...       ODS-2206    ODS-2207    ODS-1932
+    # due that our test suite is taking a long time to finish
+    # we can edit by setting an invalid bucket
+    # edit again, and set a valid bucket
+    # if the bucket is invalid, the server wont start
+    Edit S3 Data Connection    project_title=${PRJ_TITLE}    dc_name=${DC_NAME}
+    ...            aws_access_key=${S3.AWS_ACCESS_KEY_ID}    aws_secret_access=${S3.AWS_SECRET_ACCESS_KEY}
+    ...            aws_bucket_name=ods-ci-ds-pipelines-test
```

Comment:
could you move this test inside the `ods_ci/tests/Tests/400__ods_dashboard/415__ods_dashboard_projects/415__ods_dashboard_projects.robot`